Description: Depend on a lower version of AppStream
 This works around an issue in libappstream's .pc file
Forwarded: not-needed
Author: Matthias Klumpp <matthias@tenstral.net>
Last-Update: 2013-07-30

--- a/libapper/CMakeLists.txt
+++ b/libapper/CMakeLists.txt
@@ -33,7 +33,7 @@

 if(APPSTREAM)
     pkg_check_modules(GLIB2 REQUIRED glib-2.0>=2.36)
-    pkg_check_modules(APPSTREAM REQUIRED appstream>=0.3.1)
+    pkg_check_modules(APPSTREAM REQUIRED appstream>=0.3)

     set(libapper_SRCS ${libapper_SRCS} AppStream.cpp)
 endif()
